lang/src/tests/noop.rs

```diff
@@ -43,7 +43,6 @@ fn contract_compiles() {
                 use ink_core::env::{ContractEnv, EnvTypes};
 
                 pub type AccountId = <ContractEnv<DefaultSrmlTypes> as EnvTypes>::AccountId;
-                pub type AccountIndex = <ContractEnv<DefaultSrmlTypes> as EnvTypes>::AccountIndex;
```

Comment:
I guess removed in favor of just using `AccountId` ?

Reply:
AccountIndex now gone